src/clojure_sandbox/4clojure.clj

```diff
@@ -0,0 +1,176 @@
+(ns clojure-sandbox.4clojure
+  (:require [clojure.string]))
+
+;; 21. Write a function which returns the Nth element from a sequence.
+(defn nth-element [[one & others :as sequence] position]
+  (if (seq sequence)
```

Comment:
You could use when in cases where one of the branches evaluate to nil. Unless you want to be very explicit about it for some reason.

Reply:
Separately, I would check if this condition is even necessary. Because of nil punning, you may be able to avoid it.